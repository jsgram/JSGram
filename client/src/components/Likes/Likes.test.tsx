import React from 'react';
import { Likes } from './index';
import { shallow } from 'enzyme';

describe('Likes component', () => {
    let renderer: any;

    const props = {
        userId: 'some value',
        authorId: 'some value',
        loggedUsername: 'some value',
        postId: 'some value',
        authorsOfLike: [],
        loadingLike: true,
        loggedUserLikeExist: true,
        addLike: jest.fn(() => 'some value'),
        deleteLike: jest.fn(() => 'some value'),
        emitNewNotificationSocket: jest.fn(() => 'some value'),
    };

    beforeEach(() => {
        renderer = shallow(<Likes {...props} />);
    });

    test('render - success', () => {
        expect(renderer).toMatchSnapshot();
    });
});
