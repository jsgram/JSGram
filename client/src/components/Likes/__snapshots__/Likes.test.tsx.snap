// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Likes component render - success 1`] = `ShallowWrapper {}`;
